tests/Fixer/FunctionNotation/ImplodeCallFixerTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\FunctionNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Kuba Werłos <werlos@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\FunctionNotation\ImplodeCallFixer
+ */
+final class ImplodeCallFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        yield ["<?php implode('', [1,2,3]);"];
+        yield ['<?php implode("", $foo);'];
+        yield ['<?php implode($foo, $bar);'];
+        yield ['<?php $arrayHelper->implode($foo);'];
+        yield ['<?php ArrayHelper::implode($foo);'];
+        yield ['<?php ArrayHelper\implode($foo);'];
+        yield ['<?php define("implode", "foo"); implode; bar($baz);'];
+
+        yield [
+            '<?php implode("", $foo);',
+            '<?php implode($foo, "");',
+        ];
+
+        yield [
+            '<?php \implode("", $foo);',
+            '<?php \implode($foo, "");',
+        ];
+
+        yield [
+            '<?php implode("Lorem ipsum dolor sit amet", $foo);',
+            '<?php implode($foo, "Lorem ipsum dolor sit amet");',
+        ];
+
+        yield [
+            '<?php implode(\'\', $foo);',
+            '<?php implode($foo);',
+        ];
+
+        yield [
+            '<?php IMPlode("", $foo);',
+            '<?php IMPlode($foo, "");',
+        ];
+
+        yield [
+            '<?php implode("", $weirdStuff[mt_rand($min, getMac()) + 200]);',
+            '<?php implode($weirdStuff[mt_rand($min, getMac()) + 200], "");',
+        ];
+
+        yield [
+            '<?php implode("",$foo);',
+            '<?php implode($foo,"");',
+        ];
+
+        yield [
+            '<?php
+                implode(
+                    "",
+                    $foo
+                );',
+            '<?php
+                implode(
+                    $foo,
+                    ""
+                );',
+        ];
+
+        yield [
+            '<?php
+                implode(
+                    \'\', $foo
+                );',
+            '<?php
+                implode(
+                    $foo
+                );',
+        ];
```

Comment:
please add;
```php
<?php
implode(# 1
$foo/* 2.1 */,# 2.2
""# 3
);
```
and
```php
implode(# 1
# 2
$foo# 3
# 4
)# 5
;
```

Reply:
Thanks for adding the tests :)
Sorry for dropping another request, I just thought of it, can you add a test with a lot of candidates?
Something like;
```php
<?php
implode($a);implode($a);implode($a);implode($a);implode($a);implode($a);
// comment for testing
implode($a);implode($a);implode($a);implode($a);implode($a);implode($a);
```

Reply:
No problem, added.